doc/mgr/orchestrator.rst

```diff
@@ -408,6 +408,80 @@ Service Commands::
 
     ceph orch <start|stop|restart|redeploy|reconfig> <service_name>
 
+Deploying custom containers
+===========================
+
+The orchestrator enables custom containers to be deployed using a YAML file.
+A corresponding :ref:`orchestrator-cli-service-spec` must look like:
+
+.. code-block:: yaml
+
+    service_type: container
+    service_id: foo
```

Comment:
Instead of `foo`, I'd suggest to use a placeholder, e.g. `<service_identifier>` to make it clear that this needs to be changed:
```suggestion
    service_id: <service_identifier>
```